src/promptflow/promptflow/_utils/tool_utils.py

```diff
@@ -305,6 +306,33 @@ def load_function_from_function_path(func_path: str):
         )
 
 
+def assign_tool_input_index_for_ux_order_if_needed(tool):
```

Comment:
Add comments to show what is the result of this function.

Reply:
Added, thanks.